source/extensions/quic_listeners/quiche/BUILD

```diff
@@ -0,0 +1,16 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_library",
+    "envoy_package",
+)
+
+envoy_package()
+
+envoy_cc_library(
+    name = "dummy_lib",
```

Comment:
Is this just a placeholder that will be removed? If so, can you add comments and a TODO to explain this?

Reply:
done